ui-components/src/tables/TableCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { Heading } from "../headers/Heading"
+import { Button } from "../actions/Button"
+import { t } from "../helpers/translator"
+import "./TableCard.scss"
+
+export interface TableCardProps {
+  title: string
+  children?: React.ReactNode
+  elementsQty: number // to show/hide an empty state
+  onAddClick: () => void
+}
+
+const TableCard = ({ title, children, elementsQty, onAddClick }: TableCardProps) => {
```

Comment:
As a move toward generalizing our components, I'd suggest a name that doesn't rely on this component having a table as its children. Maybe just something like `Card`?

Reply:
Make sense @emilyjablonski ! I think we can follow other components and use the `StandardCard` name, what do you think?